File: Java/program60.c
///////////////////////////////////////////////////////
//
//  File name :     program60.c
//  Descreption :   Accept Input and Calculate Factors and Display
//  Author :        Vaibhav
//  Date :          20/05/2025
//
///////////////////////////////////////////////////////

#include<stdio.h>

void DisplayFactors(int iValue)
{
    int iCnt = 0;

    for(iCnt = 1;iCnt <= (iValue / 2); iCnt++)
    {
        if((iValue % iCnt) == 0)
        {
            printf("%d\n",iCnt);    
        }
    }
}

int main()
{
    int iNo = 0;
    
    printf("---- This Application is used to Display Factors of a Number----\n");

    printf("Enter Number: ");
    scanf("%d",&iNo);

    DisplayFactors(iNo);

    return 0;
}

File: program60.java
///////////////////////////////////////////////////////
//
//  File name :     program60.java
//  Descreption :   Accept Input and Calculate Factors and Display
//  Author :        Vaibhav
//  Date :          
//
///////////////////////////////////////////////////////

import java.util.*;

class Number
{
    public void DisplayFactors(int iNo)
    {
        int iCnt = 0;
        
        for(iCnt = 1;iCnt <= (iNo/2);iCnt++)
        {
            if((iNo%iCnt) == 0)
            {
                System.out.println(iCnt);
            }
        }   
    }
}

class program60
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        
        System.out.println("---- This Application is used to Display Factors of Number ----");
        System.out.println("Enter number: ");
        int iValue = sobj.nextInt();

        Number nobj = new Number();
        nobj.DisplayFactors(iValue) ;

    }
}